astro/src/content/docs/extend/examples/multi-application-dashboard.mdx

```diff
@@ -0,0 +1,166 @@
+
+Use `git clone` to clone the repository at https://github.com/fusionauth/fusionauth-example-multiapp-dashboard, or download and unzip it. 
+
+Open a terminal in the directory containing the repository files.
+
+Run the command below to start FusionAuth.
+
+  ```sh
+  docker compose up
+  ```
+
+<Aside type="note">
+  If you have completed any FusionAuth tutorials before, you might need to first delete any existing FusionAuth containers with the same name and the database volumes. Run the command below to do that.
+
+  ```sh
+  docker ps rm fa faDb; docker compose down -v
+  ```
+</Aside>
+
+Leave FusionAuth running. 
+
+In a new terminal, run the commands below to start a web server for the Changebank app, which uses FusionAuth for authentication. 
+
+  ```sh
+  cd bankApp
+  docker run --platform=linux/amd64 --rm -v ".:/app" -w "/app" node:23-alpine3.19 sh -c  "npm install"
+  docker run --platform=linux/amd64 --rm --network faNetwork -p 3000:3000 -v ".:/app" -w "/app" -e "PORT=3000" node:23-alpine3.19 sh -c  "npm run start"
+  ```
+
+The Changebank app is available at http://localhost:3000. 
+
+To learn how to make a simple Node.js app that uses FusionAuth, read the [quickstart](/docs/quickstarts/quickstart-javascript-express-web).
+
+In a third terminal, run the commands below to start a web server for a second app that uses FusionAuth for authentication. The app is called Changeinsurance, and is available at http://localhost:3001.
+
+  ```sh
+  cd insuranceApp
+  docker run --platform=linux/amd64 --rm -v ".:/app" -w "/app" node:23-alpine3.19 sh -c  "npm install"
+  docker run --platform=linux/amd64 --rm --network faNetwork -p 3001:3001 -v ".:/app" -w "/app" -e "PORT=3001" node:23-alpine3.19 sh -c  "npm run start"
+  ```
+
+Before making the dashboard, check that you can log in to all three applications. Either use an incognito browser window or don't enable [Keep me signed in](/docs/lifecycle/authenticate-users/logout-session-management#fusionauth-sso) when logging in, otherwise, you won't see the login form in the rest of this guide:
+- Browse to FusionAuth at http://localhost:9011/admin and log in with `admin@example.com` and `password`.
+  ![FusionAuth](/img/docs/extend/examples/multi-application-dashboard/fa.png)
+- Browse to Changebank at http://localhost:3000 and log in with the same username and password.
+  ![Changebank](/img/docs/extend/examples/multi-application-dashboard/changebank.png)
+- Browse to Changeinsurance at http://localhost:3001 and log in with the same username and password.
+  ![Changeinsurance](/img/docs/extend/examples/multi-application-dashboard/changeinsurance.png)
+
+If you enabled <InlineUIElement>Keep me signed in</InlineUIElement>, logging out of an application won't log you out of FusionAuth. It will only delete the session of the application. Next time you try to log in, FusionAuth will see the FusionAuth authentication cookie in your browser and automatically log you in.
```

Comment:
```suggestion
If you enabled <InlineUIElement>Keep me signed in</InlineUIElement>, logging out of an application won't necessarily log you out of FusionAuth. It will only delete the session of the application. Next time you try to log in, FusionAuth will see the FusionAuth authentication cookie in your browser and automatically log you in.
```

It will if the application redirects to FusionAuths `/oauth2/logout` URL.